guilmour-asc/rest_oai
# Address #1.

## Github issue

https://github.com/guilmour-asc/rest_oai/issues/1

## What this PR does

Renames src/Encoder/OAIXMLEncoder.php to src/Encoder/OaiXmlEncoder.php.

## How to test

On a clean Drupal 8 install:

1.  In the master branch, install the module using the Drupal admin GUI
1. You should get a WSOD
1. Uninstall the module using drush: `drush pmu rest_oai`
1. Check out the issue-1 branch
1. Install the module using the Drupal admin GUI
1. All should go well.
